user/lib.rs

```diff
@@ -0,0 +1,19 @@
+use std::sync::Arc;
+
+use system::concepts::{Credential, PasswordHash, User};
+
+use crate::user_manager::UserManager;
+
+pub mod errors;
+pub mod user_manager;
+
+pub fn initialise_default_user(user_manager: &Arc<UserManager>) {
+    if !user_manager.contains("admin") {
```

Comment:
These constants should go in `constants.rs`

Reply:
Fixed in b9c8e80f8260a7b32a597e6374418e00cdf7d5d7